torch/_prims/nvfuser_executor.py

```diff
@@ -366,7 +374,7 @@ def maybe_partition_graph(
 
 class NVTXInterpreter(torch.fx.Interpreter):
     def run_node(self, n):
-        torch.cuda.nvtx.range_push("name: {0}, args: {1}, op: {2}, meta: {3}".format(n.name, n.args, n.op, n.meta))
```

Comment:
Did you want to keep `n.meta` or removing it is intentional?

Reply:
Intentionally removed it, that was only for a local experiment